content/questions/foundations/question-109.md

```diff
@@ -0,0 +1,18 @@
+---
+archetype: "questions"
+title: "Question 109"
+question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub?"
+draft: false
+---
+
+> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
+- [x] Passkey
+- [x] Text message
+- [ ] Phone call
+> While text messages are supported in some countries, there is no mention of phone calls for 2FA/MFA.
+- [ ] Email
+> There is no mention of email as an option to receive a code for MFA/2FA.
```

Comment:
Don't assume that people actually will click and read the links in the light bulb - that would be cheating on the actual exam. Therefore explanations shouldn't include "there is no mention in .."
```suggestion
```